entity-resolution/python/entity-resolution.py

```diff
@@ -8,7 +8,7 @@
 from pslpython.rule import Rule
 
 MODEL_NAME = 'entity-resolution'
-DATA_DIR = os.path.join('..', 'data', MODEL_NAME)
+DATA_DIR = os.path.join('..', 'data', MODEL_NAME, '0')
```

Comment:
Make the split a constant so users can change it.

Reply:
I've added another constant called MODEL_SHORT_NAME.